app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -53,16 +53,16 @@ class ChatActivity : AppCompatActivity(), IChatView {
     val TAG: String = ChatActivity::class.java.name
 
     lateinit var chatPresenter: IChatPresenter
-    val PERM_REQ_CODE = 1
-    lateinit var recyclerAdapter: ChatFeedRecyclerAdapter
+    private val PERM_REQ_CODE = 1
+    private lateinit var recyclerAdapter: ChatFeedRecyclerAdapter
     var textToSpeech: TextToSpeech? = null
     var recordingThread: RecordingThread? = null
```

Comment:
private?

Reply:
@iamareebjamal As these members are being used only within this activity and could be made private, therefore I used the private access modifier here.

Reply:
I meant why not the variables on review line are private?

Reply:
@iamareebjamal Made textToSpeech private. But, recordingThread has not been made private because it is being used in STTfragment.kt .

<img width="1059" alt="screen shot 2018-05-31 at 10 19 33 am" src="https://user-images.githubusercontent.com/30979369/40762021-27034fa8-64bc-11e8-8ea8-7c730665319d.png">